openreview/venue/group.py

```diff
@@ -433,6 +433,29 @@ def create_ethics_chairs_group(self):
                 ethics_chairs_group.web = content
                 self.post_group(ethics_chairs_group)
 
+    def create_publication_chair_group(self):
+        venue_id = self.venue_id
+
+        publication_chair_group_id = self.venue.get_committee_id('Publication_Chair')
```

Comment:
I think the group name should be in plural, there can be more than one publication chair.